cunumeric/module.py

```diff
@@ -1177,6 +1182,239 @@ def squeeze(a: ndarray, axis: Optional[NdShapeLike] = None) -> ndarray:
     return a.squeeze(axis=axis)
 
 
+def broadcast_shapes(*args: Sequence(Union[tuple(int), int])) -> tuple(int):
+    """
+
+    Broadcast the input shapes into a single shape.
+
+    Parameters
+    ----------
+    `*args` : tuples of ints, or ints
+        The shapes to be broadcast against each other.
+
+    Returns
+    -------
+    tuple : Broadcasted shape.
+
+    See Also
+    --------
+    numpy.broadcast_shapes
+
+    Availability
+    --------
+    Multiple GPUs, Multiple CPUs
```

Comment:
I don't think this implementation will work on GPUs or multiple CPUs. I would change this to the `Single CPU`

Reply:
Done